diff --git a/rkbin/tools/setting.ini b/rkbin/tools/setting.ini
index 045ea6c..1f27dfd 100644
--- a/rkbin/tools/setting.ini
+++ b/rkbin/tools/setting.ini
@@ -5,7 +5,7 @@ hard_sign_litte_hash= 3399|3328
 hard_sign_pss= 3562|3528|3588|3566|3568|3308|3326|px30|1808|3228P|1109|1126|2206|1106
 new_crypto= 3562|3528|3588|3566|3568|3308|3326|px30|1808|1109|1126|2206|1106
 new_idb= 3562|3528|3588|3566|3568|1106
-sign_flag=
+sign_flag=0x20
 sign_soft_version=
 sign_nonce=
 sign_cert=
